0001_cars_test.js

```diff
@@ -291,9 +254,9 @@ describe("Test Child:", () => {
       // TODO: shouldn't need to wait and reload.
       // cy.visit("/");
 
-      cy.get(".webix_progress_state.wxi-sync.webix_spin").should(
-         "not.be.visible"
-      );
+      cy.get(".webix_progress_state.wxi-sync.webix_spin", {
+         timeout: 30000,
```

Comment:
You can increase your local "defaultCommandTimeout" to a higher number to avoid making test changes to get this to work.